### PR TITLE
[OSDOCS-12718] Support A3 instances with NVIDIA H100 80GB GPUs

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.17+
Issue:

https://issues.redhat.com/browse/OSDOCS-12718
Link to docs preview:

https://85178--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-service-definition.html#gcp-compute-types_osd-service-definition
QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by svmrh** [comment]
Looks good. Thanks for putting this together @mletalie . 
One small change. We need to include a3 in this note:

> e2 and a2 compute types are available for CCS only.

**Comment by mletalie** [comment]
> Looks good. Thanks for putting this together @mletalie . One small change. We need to include a3 in this note:
> 
> > e2 and a2 compute types are available for CCS only.

Thanks, good catch, @svmrh.

File: modules/sdpolicy-am-gcp-compute-types.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@
 {product-title} offers the following worker node types and sizes on Google Cloud that are chosen to have a common CPU and memory capacity that are the same as other cloud instance types:
 [NOTE]
 ====
-`e2` and `a2` compute types are available for CCS only.
+`e2`, `a2` and `a3` compute types are available for CCS only.
```
Comment (aspauldi) [inline modules/sdpolicy-am-gcp-compute-types.adoc:11]: IBM Style Guide advises using a comma to separate items in a series of three or more (https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series. 
```suggestion
`e2`, `a2`, and `a3` compute types are available for CCS only.
```

**Review by aspauldi — COMMENTED** [review]
Hi, Just one comment. Otherwise, looks good to me!

**Comment by lento-sun** [comment]
/lgtm 
compared with info in https://cloud.google.com/compute/docs/accelerator-optimized-machines#a3-vms

**Pull request merged**